synapse/types/__init__.py

```diff
@@ -769,7 +772,7 @@ def copy_and_replace(self, key: str, new_value: Any) -> "StreamToken":
         return attr.evolve(self, **{key: new_value})
 
 
-StreamToken.START = StreamToken(RoomStreamToken(None, 0), 0, 0, 0, 0, 0, 0, 0, 0)
+StreamToken.START = StreamToken(RoomStreamToken(None, 0), 0, 0, 0, 0, 0, 0, 0, 0, 0)
```

Comment:
To check: when we first write to the un-partial state rooms stream, does that write occur with position 1?

Reply:
The associated postgres sequence seems to be [initialized to 1](https://github.com/matrix-org/synapse/blob/develop/synapse/storage/schema/main/delta/73/21_un_partial_stated_room_stream_seq.sql.postgres#L19) so I would say yes.

Reply:
Or [even 2](https://github.com/matrix-org/synapse/pull/13766) so we should be safe :)

Reply:
Thanks for checking. (I vaguely remember we've had problems with this sort of thing before.)